# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/ScriptKind.h
# Commit: 85fd9fc9cffa
# Full Hash: 85fd9fc9cffa98039328796d30a2d10386283b3e
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-21 21:39:46
# Regressor Bug: 1742437
# File Overlap Count: 2
# Description:
#   Bug 1742437 - Move module loader files into a central folder; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137532
# ==============================================================================

diff -r 7ecc4f9b20ad -r 85fd9fc9cffa dom/script/ScriptKind.h
--- a/dom/script/ScriptKind.h	Mon Feb 21 15:39:08 2022 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,18 +0,0 @@
-/* -*- Mode: C++; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* vim: set ts=8 sts=2 et sw=2 tw=80: */
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#ifndef mozilla_dom_ScriptKind_h
-#define mozilla_dom_ScriptKind_h
-
-namespace mozilla {
-namespace dom {
-
-enum class ScriptKind { eClassic, eModule, eEvent };
-
-}  // namespace dom
-}  // namespace mozilla
-
-#endif